luxmed_api.py

```diff
@@ -56,7 +58,6 @@ def _prepare_headers(self, api_version: str):
         access_token = self._get_access_token()
 
         return {"Authorization": "Bearer " + access_token,
-                "Api-Version": api_version,
```

Comment:
`_prepare_headers()` function has unused parameter now, should be removed as well